starknet_devnet/blueprints/rpc/classes.py

```diff
@@ -64,10 +64,11 @@ async def get_class_at(block_id: BlockId, contract_address: Address) -> dict:
     await assert_block_id_is_valid(block_id)
 
     try:
-        result = await state.starknet_wrapper.get_class_by_address(
+        result_dict = await state.starknet_wrapper.get_class_by_address(
```

Comment:
I'm not sure about hungarian notation in Python because I'm not python expert, but just saying that it might be not necessary to use it